xdsl/irdl/irdl.py

```diff
@@ -224,13 +225,9 @@ def attr_constr_coercion(
     Attributes are coerced into EqAttrConstraints,
```

Comment:
Should we just delete this function, and replace it with `irdl_to_attr_constraint`?

Reply:
Probably!